### DISCUSSION TITLE
i want to use csv helper to read the column headers which will be changing Scenario1:Identifier,data shares,records,pointer heads scenario2:IDENTIFIER,Data Shares,Records,Pointer heads.Is there any way to convert these headers to Upper case and use it

### DISCUSSION THREAD

**Discussion by saireddy111** [JoshClose/CsvHelper#1318] [Q&A]
All questions should be directed to StackOverflow. https://stackoverflow.com/questions/tagged/csvhelper

Please only submit a question if no one has been able to answer your question on StackOverflow. I have very limited time to work on this project, and would prefer to use that time for bugs and features.

Thanks.

**Comment by jzabroski** [comment]
@saireddy111 Did you ask this question on StackOverflow first?
